src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -42,6 +43,7 @@ export const InviteMemberDialogContent = ({
     if (!result.ok) return;
 
     setIsLinkButtonClicked(true);
+
     const hash = result.value;
 
     return navigator.clipboard.writeText(CLIENT_BASE_URL + '/groups/join/' + hash);
```

Comment:
`return` should be in a new line because the return value is not a simple void but Promise<void>, and it could be misleading. 
Since the type of the return value is not specified in the TSDoc, it should always be `void`, which means returns nothing.

<img width="539" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/17cb0527-426a-4687-8d25-78baefd412f8">

The better version would be like this:
```ts
await navigator.clipboard.writeText(CLIENT_BASE_URL + '/groups/join/' + hash);
return;
```